src/core/link.js

```diff
@@ -205,7 +205,9 @@
          * @return {String} The URI updated with the protocol.
          */
         _getCompleteURI: function(URI) {
-            if (!REGEX_URI_SCHEME.test(URI)) {
+            if (URI.indexOf('@') && URI.match(/[^\/]+@/)) {
```

Comment:
The whole change is not documented in function's comment part above. There is currently only `If not, the default 'http' scheme`, but with this change this is no more true.